### PR TITLE
Update prettier.yml

### PR DESCRIPTION
**Pull request by danielkcz**

### CODE REVIEW

**Review by danielkcz — COMMENTED** [review]
fooooooooo

**Review by danielkcz — COMMENTED** [review]
baz

**Review by danielkcz — COMMENTED** [review]
now?

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 name: "Run Prettier for PR"
 on:
-  - pull_request
+  pull_request_review: {}
+  pull_request_review_comment: {}
```
Comment (danielkcz) [inline .github/workflows/prettier.yml:4]: Come on, work you beast.

**Review by mweststrate — APPROVED** [review]
Blindly approving this one :P

**Pull request closed without merging**